phpBB/assets/javascript/editor.js

```diff
@@ -226,6 +231,55 @@ function addquote(post_id, username, l_wrote) {
 	return;
 }
 
+/**
+* Create a quote block for given text
+*
+* Possible attributes:
+*   - author:  author's name (usually a username)
+*   - post_id: post_id of the post being quoted
+*   - user_id: user_id of the user being quoted
+*   - time:    timestamp of the original message
+*
+* @param  {!string} text       Quote's text
+* @param  {!Object} attributes Quote's attributes
+* @return {!string}            Quote block to be used in a new post/text
+*/
+function generate_quote(text, attributes)
+{
+	var quote = '[quote';
+	if ('author' in attributes)
+	{
```

Comment:
Parenthesis shouldn't be on a new line in javascript according to our coding guidelines